doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -77,6 +77,8 @@ this view.
 | `redacted_sql`            | [`text`]                     | The SQL text of the statement, in a normalized form, with all string and numeric literals hidden.                                                                                                                                                                             |
 | `prepared_at`             | [`timestamp with time zone`] | The time at which the statement was prepared.                                                                                                                                                                                                                                 |
 | `statement_type`          | [`text`]                     | The _type_ of the statement, e.g. `select` for a `SELECT` query, or `NULL` if the statement was empty.                                                                                                                                                                        |
+| `n_throttled`             | [`uint8`]                    | The number of statements that were dropped due to throttling before this one was seen. If you have a very high volume of queries and need to log them without throttling, please contact support.                                                                             |
```

Comment:
`uint8`s are not understandable by many drivers because it's not a native postgres oid type. I encourage the use of int8 here instead.

Reply:
We're already using `uint8` for various other things in this view (at least `execution_timestamp` and `transaction_id`). I didn't want to introduce an inconsistency for just this one field -- maybe we should consider switching those all over to `int8` at the same time.

Btw, do you know anything about why we support `uint8` at all, given that it's not commonly supported?